src/OutputWriters/output_writer_utils.jl

```diff
@@ -52,6 +52,23 @@ function update_file_splitting_schedule!(schedule::FileSizeLimit, filepath)
     return nothing
 end 
 
+# Update schedule based on user input
+is_output_splitted!(schedule, filepath) = nothing
+
+function is_output_splitted!(schedule::TimeInterval, filepath, overwrite_existing) 
```

Comment:
I think my only comment is that I find this function name confusing. `is_output_splitted!` suggests it will return a boolean `true` or `false`, but it actually returns a part number and a filepath?